src/main/java/frc/robot/RobotContainer.java

```diff
@@ -37,14 +38,14 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-  private final Swerve swerveDrive = Swerve.getInstance();
+  private final Swerve swerve;
   private final SendableChooser<SequentialCommandGroup> autoChooser;
   private final AutoCommands autos;
 
-  private final Hood hood = Hood.getInstance();
-  private final Shooter shooter = Shooter.getInstance();
-  private final Indexer mIndexer = Indexer.getInstance();
-  private final Intake mIntake = Intake.getInstance();
+  private final Hood hood;
+  private final Shooter shooter;
+  private final Indexer indexer;
+  private final Intake intake;
 
   private final XboxController driver = new XboxController(SushiConstants.OI.DRIVER_PORT);
   private final XboxController op = new XboxController(1);
```

Comment:
or we are doing inline initialization???

Reply:
nah initialization in constructor this is in style guide